public/app/features/explore/TraceView/components/TracePageHeader/TracePageHeader.tsx

```diff
@@ -176,20 +196,78 @@ export default function TracePageHeader(props: TracePageHeaderEmbedProps) {
     return { ...rest, value: renderer(trace, timeZone, styles) };
   });
 
+  const spanWithHeaderTags = getSpanWithHeaderTags(trace.spans);
+
   const title = (
     <h1 className={styles.TracePageHeaderTitle}>
       <TraceName traceName={getTraceName(trace.spans)} />{' '}
       <small className={cx(styles.TracePageHeaderTraceId, uTxMuted)}>{trace.traceID}</small>
     </h1>
   );
 
+  const newTitle = (
+    <h1 className={cx(styles.TracePageHeaderTitle)}>
+      <TraceName traceName={getTraceName(trace.spans)} />
+      <small>
+        <span className={styles.divider}>|</span>
+        {formatDuration(trace.duration)}
+      </small>
+    </h1>
+  );
+
+  const method = spanWithHeaderTags?.tags.filter((tag) => {
+    return tag.key === 'http.method';
+  });
+
+  const status = spanWithHeaderTags?.tags.filter((tag) => {
+    return tag.key === 'http.status_code';
+  });
+
+  let statusColor: BadgeColor = 'orange';
+  if (status && status.length > 0 && Number.isInteger(status[0].value)) {
```

Comment:
This coloring looks a bit weird to me. Seems like 4xx will be red while 5xx will be orange? Would expect 5xx = red, 4xx = orange, and everything else green.

Reply:
So should we change this? I mean 5xx errors are usually more severe than 4xx. 4xx can mean user just created wrong request but 5xx usually means error in your application. Also not for example 1xx or 3xx should be green but seems like we default to orange now.

Reply:
I think we should and I changed it last week. This is what we have:

`let statusColor: BadgeColor = 'green';`
`if (status && status.length > 0 && Number.isInteger(status[0].value)) {`
    `  if (status[0].value.toString().charAt(0) === '4') {
      statusColor = 'orange';
    }` 
   `  else if (status[0].value.toString().charAt(0) === '5') {
      statusColor = 'red';
    }`
`}`

Reply:
yep that looks good